File: max.rb
print "配列の最大値を求めます\n"
p arr = [4,2,34,67,12,5]
i = 1
max = arr[0]

while i < arr.size 
  if max < arr[i]
    max = arr[i] 
  end
  i += 1
end

print max , "\n"

File: max.c
#include "stdio.h"

int main(void) {
	int data[5];
	int i, max;

	scanf("%d", &data[0]);
	max = data[0];
	
	for (i = 1; i < 5; i++) 
	{
		scanf("%d", &data[i]);
		if (max < data[i])
		{
			max = data[i];
		}
	}

	
	printf("%i\n", max);
}